src/connectors/hits-per-page/connectHitsPerPage.ts

```diff
@@ -259,11 +265,14 @@ You may want to add another entry to the \`items\` option with this value.`
       },
 
       getWidgetRenderState({ state, results, createURL, helper }) {
+        const hasNoResults = results ? results.nbHits === 0 : true;
+
         return {
           items: transformItems(normalizeItems(state), { results }),
           refine: connectorState.getRefine(helper),
           createURL: connectorState.createURLFactory({ state, createURL }),
-          hasNoResults: results ? results.nbHits === 0 : true,
+          hasNoResults,
+          canRefine: !hasNoResults,
```

Comment:
Since we're deprecating `hasNoResults` and planning on remove it, wdyt of reversing the logic so `hasNoResults` is the inverse of the computed value for `canRefine`? I understand it might not be easy for the other connectors since there are additional checks, but here it could work.

Reply:
Makes sense, thanks ! :)